src/pages/UserPage/_user_page.scss

```diff
@@ -0,0 +1,110 @@
+@import '../../utils/scss/constants';
+
+.user-page {
+  width: 100vw;
```

Comment:
Tutaj zamiast width 100vh dałbym max-width 1366px i następnie margin 0 auto, bo jeśli ktoś otworzy stronę na dużym monitorze to rozciągnie się na całej szerokości i trzeba za dużo kręcić głową :).
Jak dasz max-width i margin auto po bokach to będzie zawsze wyśrodkowane.

width : 100vh
![Zrzut ekranu (27)](https://user-images.githubusercontent.com/101992103/234176782-dc08176c-d44d-4169-8934-2ab1e7e23307.png)

max-width 1366px i margin 0 auto
![Zrzut ekranu (26)](https://user-images.githubusercontent.com/101992103/234176650-ccbd5b31-1744-4b95-953c-ecd52722f909.png)